index.ts

```diff
@@ -270,13 +270,18 @@ class IGCParser {
   private parseARecord(line: string): IGCParser.ARecord {
     let match = line.match(RE_A);
     if (!match) {
+      if (line.match(/^A\w{3}/)) {
+        const manufacturer = lookupManufacturer(line.substr(1, 3));
+        const additionalData = line.length > 4 ? line.substr(4) : null;
+        return { manufacturer, additionalData, numFlight: null, loggerId: '' };
```

Comment:
`loggerId` is `""` instead of null so that `IGCParser.ARecord` does not have to be modified (backward compatibility)